samples/metrics/tracing/Metrics_1/Endpoint/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.Metrics.Tracing.Endpoint";
+        var endpointConfig = new EndpointConfiguration("Samples.Metrics.Tracing.Endpoint");
+        endpointConfig.SendFailedMessagesTo("error");
+        endpointConfig.UsePersistence<InMemoryPersistence>();
+        endpointConfig.EnableInstallers();
+
+        #region EnableMetricTracing
+
+        var metricsOptions = endpointConfig.EnableMetrics();
+        metricsOptions.EnableMetricTracing(TimeSpan.FromSeconds(5));
+
+        #endregion
+
+        var endpointInstance = await Endpoint.Start(endpointConfig)
+            .ConfigureAwait(false);
+
+        try
+        {
+            Console.WriteLine("Endpoint Started. Press any key to send a message");
```

Comment:
Most other samples use a convention of: Press Enter to do something. Press any other key to exit.